index.js

```diff
@@ -25,6 +25,9 @@ function levelgraphJSONLD(db, jsonldOpts) {
     var blanks = {};
 
     jsonld.toRDF(obj, options, function(err, triples) {
+      if (err || triples.length === 0) {
```

Comment:
If there's no triples, there's nothing more to do, and err is undefined, so it should all work out.
It's not strictly necessary I guess, I just copy-pasted it from fetchExpandedTriples lower down in the file which does the same thing.